app/config/projects.yml

```diff
@@ -462,6 +462,32 @@ parameters:
                     current: true
                     aliases: ['current', 'stable']
 
+        coding-standard:
+            active: true
+            name: Coding Standard
+            shortName: Coding Standard
+            slug: coding-standard
+            docsSlug: doctrine-coding-standard
+            composerPackageName: doctrine/coding-standard
+            repositoryName: coding-standard
+            docsRepositoryName: coding-standard
+            docsPath: /docs
+            description: >
+                The Doctrine Coding Standard is a set of PHPCS rules applied to all Doctrine projects.
+
+            keywords: [coding, standard, rules, sniffs, checks]
+            versions:
+                -
+                    name: '5.0'
+                    branchName: 'master'
```

Comment:
Wondering about the maintainability of this: would it make sense to use the `branch-alias` feature of composer to figure these out? To be more clear, composer only looks at `master` when finding branch aliases

Reply:
It would be nice to have something more advanced here. But yes, currently as soon as we release 5.0 and master becomes 6.0, we'd have to change this from master to 5.x or whatever we name the branch.